### PR TITLE
Separate tests replased with one script in examples CI

### PR DESCRIPTION
**Pull request by volovyks**
We are forgetting to add new tests to CI, let's simplify logic a bit.

### CODE REVIEW

**Comment by volovyks** [comment]
@itegulov after the merge of ABI I noticed that some tests are failing locally (tests that were not added to the CI). Can you please check what is wrong with them?

**Comment by itegulov** [comment]
https://github.com/near/near-sdk-js/pull/322 should resolve the ABI generation errors

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -23,32 +23,6 @@ jobs:
...
-      - name: parking-lot
-        run: cd examples && pnpm build:parking-lot && pnpm test:parking-lot
-      - name: standard nft
-        run: cd examples && pnpm build-nft && pnpm test:nft
-      - name: programmatic-update
-        run: cd examples && pnpm build:programmatic-update && pnpm test:programmatic-update
-      - name: cross-contract-call-loop
-        run: cd examples && pnpm build:cross-contract-call-loop && pnpm test:cross-contract-call-loop
+        run: pnpm install
+      - name: Run exempes tests
```
Comment (ailisp) [inline .github/workflows/examples.yml]:
```suggestion
      - name: Run examples tests
```

**Review by ailisp — APPROVED** [review]
Nice! Looks good to me!

**Pull request merged**